.github/workflows/run-e2e-tests.yaml

```diff
@@ -0,0 +1,135 @@
+name: Run TSPS e2e tests with BEE
+on:
+  pull_request:
```

Comment:
this is just here to test in this branch. the `schedule` below is probably what we want to do instead.  Lets chat about what that schedule should be